osclib/stagingapi.py

```diff
@@ -1197,6 +1208,11 @@ def adi_prj_from_number(self, number):
             return number
         return '{}:adi:{}'.format(self.cstaging, number)
 
+    def adi_prj_from_adi_name(self, name):
```

Comment:
Use `prj_from_short()` instead. `short` means short-hand names like: `A` or `adi:17` which is what you want.